src/navigation/effects.js

```diff
@@ -306,7 +306,9 @@ export const emojiPreset = {
         <View
           style={{
             alignItems: 'center',
-            top: HeaderHeightWithStatusBar,
+            // See: src/components/profile/ProfileMasthead.js. <AvatarCircle />
+            // has marginTop: 10.
+            top: HeaderHeightWithStatusBar + (android ? 10 : 0),
```

Comment:
Well i would exact it into a constant because then it's very easy to actually change in one place and break in another